### PR TITLE
Allows for duplicate questions, added Bootstrap to questions page

### PR DESCRIPTION
**Pull request by haolinzhu**
- Questions no longer removed when dragged to Selected Questions
- Allows each question to be selected multiple times
- Added Bootstrap to questions page
- TO-DO: option to remove a selected question

### CODE REVIEW

**Review by ToxicSam — APPROVED** [review]
looks good

File: app/views/questions/index.html.erb [hunk 1]
```diff
@@ -5,46 +5,69 @@
...
-          </select>
-          <label class='question_title'>Credit:</label>
-          <input type="text" value="100" name="assessment_credit" size=4><br><br>
-
+          <div class="form-group w-75">
+            <label class='question_title'>Assessment name:</label>
+            <input class="form-control" type="text" placeholder="Enter assessment name" size=36>
+          </div>
+          <div class="form-row">
+            <div class="col-md-5 mb-3">
```
Comment (kmoy1) [inline app/views/questions/index.html.erb:30]: Could use spacing between column tags for better visibility

**Review by kmoy1 — COMMENTED** [review]
**What is the goal of this PR?**
Make the visual appearance nicer and more abstracted by implementing bootstrap + add a duplicate questions feature to UI. 

**Were you able to view screenshots, story cards, and or test the app?**
Yes. The app seems to load correctly and implement duplicate features. Visual screenshots were available on the info.yml file. 

**Is the PR in a shape that makes it easy to review?**
It was very easy to review. Commits were given clear descriptions and followed a logical structure. Added HTML elements and javascript functions were given clear names. Individual code edits were modular and not overly complex/lengthy.

**What are the status of the tests?**
Are the tests included all passing? Are the tests comprehensive? Are there any features that need to be tested? Are there any tests that are redundant?

There were no tests included. The functionality of the site with some web driver (Selenium) should be implemented, and the duplicate question feature should be tested as well. 

**What is the style of the code like?**
Are there any code linters that were run automatically?
What is their status?
If there are no linters, you get to play the human code linter.

No code linters (according to the Travis build configuration file). Travis indicates the build passed with this PR, though.  However, style seems to be fine, with the exception of lack of spacing between HTML elements (which may/may not be nitpicky).

**Is there anything missing from this PR that should be included?**
**Testing for features implemented.** 

Comments didn't seem as necessary for a PR such as this one, as it was very easy to tell what added code corresponded to in terms of parts needed to complete the feature. 

**Feature Design: Are there any parts of this feature that are cause for concern?**
Nothing in regards to functionality or dangerous lines of code written. 

### Overall Assessment

**What does this PR do well?**
Code is consistent, easy to understand, and not overly lengthy. Commits were not overly lengthy and were concisely representative of the PR's goal without any extra files/comments/notes/etc. 

**What is something that could be improved with this PR?**
Include testing for the duplicate questions feature implemented. Additionally, spacing between html tags such as rows/cols/forms might make the code more readable. 

Also, it kind of focuses on two different things at once (one styling and one functionality)- it might have better to split into separate PRs, for rollback purposes.

File: app/views/questions/index.html.erb [hunk 2]
```diff
@@ -13,38 +18,57 @@
...
+            </div>
+            <label class='question_title'>End date:</label>
+            <div class="form-row">
+              <div class="col-md-5 mb-3">
+                <input class="form-control" type="date" min="2021-03-02" max="2100-12-31">
+              </div>
+              <div class="col-md-5 mb-3">
+                <input class="form-control" type="time" min="00:00" max="23:59">
+              </div>
+            </div>
```
Comment (tsrivatsav) [inline app/views/questions/index.html.erb:68]: Not really sure why the start date and end date have the same max but that's probably just an error in my understanding.

File: app/views/questions/index.html.erb [hunk 3]
```diff
@@ -64,77 +88,53 @@
...
+        var spantag = document.createElement("span");
+        spantag.innerHTML = "&times;";
+        spantag.setAttribute('aria-hidden', 'true');
+        closetag.setAttribute('aria-label', 'Close');
+        closetag.classList.add("close");
+        closetag.type = "button";
+        closetag.appendChild(spantag);
+        tag.append(closetag);
+        $(this).append(tag);    
+      }
```
Comment (tsrivatsav) [inline app/views/questions/index.html.erb:136]: This whole section seems like it has the potential to be done in a cleaner way. Maybe check the API to see if there's a function that can add attributes to classList in one line
